src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -111,28 +113,49 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
   }
 
   async componentDidMount() {
+    disposeOnUnmount(this, [
+      namespaceStore.onContextChange(() => this.loadStores(), {
+        fireImmediately: true,
+      })
+    ]);
+  }
+
+  async componentWillUnmount() {
+    this.isUnmounting = true;
+    await when(() => this.isLoaded);
```

Comment:
This seems unnecessary, especially since JS is single threaded.

Reply:
Well, this is partially true imo.. Just removing this would be not enough.. 
Consider a case:

1. `componentWillUnmount` called in the middle of stores loading (e.g. ns context changed) => unsubscribe is called
2. last actively loading store has finished and it adds own subscription
3. next iteration cycle reaches `break`, nothing new is loading but we have a stale subscription from previous step..

So I've added extra `unsubscribeStores()` when cycle is broken.